python_tests/test_python_frontend.py

```diff
@@ -20,6 +20,7 @@
 import torch._refs as refs
 import torch._prims as prims
 
+import nvfuser
```

Comment:
I wanted the call to be `nvfuser.serialize`, which would distinguish it from `fusion_cache.serialize`. I thought that `serialize` is a common function name and there could be potential name conflicts.

Reply:
Got ya. I'm fine with it as-is, since this is only testing scripts.

Alternatively, something with this can avoid conflicts. i.e.
```
from nvfuser import (
  serialize as nv_serialize,
)
```

Reply:
Good idea! I forgot you could do that in python.